### ISSUE TITLE
Source data of Deng&Ryu's dataset

### ISSUE THREAD

**Issue by xiaomingaaa** [Zhankun-Xiong/MRCGNN#4]
Hello,

Only the five groups' DDI data are available, could you provide the entire DDI data (the dataset used in Table 2 for your paper)?

**Comment by Zhankun-Xiong** [comment]
> Hello,
> 
> Only the five groups' DDI data are available, could you provide the entire DDI data (the dataset used in Table 2 for your paper)?

Hi, sorry for the late reply, in fact, you can get the entire DDI data by merging the training set, validation set and the test set in either group. That is to say, we only divided the entire DDI data to different groups and did nothing else.
